plugins/plugin_manager/init.lua

```diff
@@ -359,4 +366,9 @@ if pcall(require, "plugins.terminal") then
   })
 end
 
+keymap.add({
+  ['ctrl+shift+1'] = 'plugin-manager:show',
+  ['ctrl+shift+2'] = 'plugin-manager:open-session'
+})
+
```

Comment:
Added for testing?

Reply:
Nah; should actually be a change on master (but whatever, may as well add it here; gives an easy way to open the plugin manager. Think I should nix it?

Reply:
Yep